### PR TITLE
Sm/repro gh 883

### PR DESCRIPTION
**Pull request by mshanemc**
### What does this PR do?
fix userId on user:create

### What issues does this PR fix or reference?
@W-9036859@
https://github.com/forcedotcom/cli/issues/883

### CODE REVIEW

File: test/allCommands.nut.ts [hunk 1]
```diff
@@ -75,6 +79,7 @@ describe('verifies all commands run successfully ', () => {
       'lastname',
       'timezonesidkey'
     );
+    expect(output.result.fields.id).to.not.equal(mainUserId);
```
Comment (WillieRuemmele) [inline test/allCommands.nut.ts:82]: should we also be verifying that it equals whatever is printed or whatever it should be?

Comment (mshanemc) [reply test/allCommands.nut.ts:82]: the NUT will generate a user with an auto-generated ID.  Are you wanting to assert something about the length and first 3 characters?

This assertion is really just that the bug is fixed.

File: test/allCommands.nut.ts [hunk 2]
```diff
@@ -29,6 +31,7 @@ describe('verifies all commands run successfully ', () => {
         `sfdx force:org:create -d 1 -s -f ${path.join('config', 'project-scratch-def.json')}`,
         'sfdx force:source:push',
       ],
+      retries: 2,
```
Comment (peternhale) [inline test/allCommands.nut.ts:34]: @mshanemc test session retries should be delegated to circle

File: test/allCommandsNoJSON.nut.ts [hunk 3]
```diff
@@ -23,6 +23,7 @@ describe('verifies all commands run successfully (no json)', () => {
         `sfdx force:org:create -d 1 -s -f ${path.join('config', 'project-scratch-def.json')}`,
         'sfdx force:source:push',
       ],
+      retries: 2,
```
Comment (peternhale) [inline test/allCommandsNoJSON.nut.ts:26]: @mshanemc same here - delegate to circle

File: test/commands/user/create.nut.ts [hunk 4]
```diff
@@ -26,6 +26,7 @@ describe('creates a user from a file and verifies', () => {
         `sfdx force:org:create -d 1 -s -f ${path.join('config', 'project-scratch-def.json')}`,
         'sfdx force:source:push',
       ],
+      retries: 2,
```
Comment (peternhale) [inline test/commands/user/create.nut.ts:29]: same

**Review by peternhale — CHANGES_REQUESTED** [review]

**Review by peternhale — APPROVED** [review]

**Review by WillieRuemmele — APPROVED** [review]

File: test/allCommands.nut.ts (hunk 1 above)
Comment (shetzel) [reply test/allCommands.nut.ts:81]: Looks like the unit test covers that nicely as well.  👍

**Review by shetzel — APPROVED** [review]

**Review by RodEsp — APPROVED** [review]
This is pretty cool.

**Pull request merged**